### ISSUE TITLE
Extension dosen't load anymore

### ISSUE THREAD

**Issue by Ostpopcorn** [microsoft/vscode-cpptools#7080]
Type: General

**Describe the bug**
- OS and Version: Windows 10 (x64) v1909
- VS Code Version: 1.53.2 
- C/C++ Extension Version: Upgrade from 1.1.x to 1.2.2
- Other extensions you installed (and if the issue persists after disabling them): 
This appeard after starting VSC today and another extension (vscode-esp-idf-extension) did not start, after a issue there I found that it was caused by cpptools did not load.
![bild](https://user-images.githubusercontent.com/18281488/109505076-87659380-7a9c-11eb-8502-7efa0f94f93e.png)
There and then i decided that I would try to reinstall cpptools and changeing what version would be installed but the same persisted. Then with a swift stroke i removed all folder in extensions that had any "cpptools" in their name and now i got this error:
![bild](https://user-images.githubusercontent.com/18281488/109506686-3787cc00-7a9e-11eb-9692-fe33a0e31d88.png)
Pressing "log.ts:191" directs me to ConsoleLogService class from where just the console.log was...

**Expected behavior**
Well it should be loaded correctly. 

**Additional context**
Don't think I have opened VSC i a little while (~1 month) so maybe something happend in the update. But to be clear I have used it without problem for a long time, so it's not a fresh install. 

If there is something else you would like see I'll share as soon as possible.

**Comment by Colengms** [comment]
Hi @Ostpopcorn .  If you are seeing a "Cannot find module" error, it's possible an error occurred when installing/upgrading the extension. I had opened this issue against VS Code for previous occurrences of this: microsoft/vscode#100537

Could you take a look at that issue, and set aside the vsix they are asking for? Then, could you try deleting the entire ms-vscode.cpptools-x.x.x folder from the extensions folder and reinstalling the extension? If the newly installed version works, perhaps we could get VS Code to revisit that issue.

**Comment by Ostpopcorn** [comment]
Hi @Colengms! 
The "did not find module" was before I deleted the folder for the extension the first time, and I dont think that can be reversed now. 
But now I did the following and no change.

1. Reinstall without removing any folder
2. Reinstall + removing extension folder
3. Reinstall + removing extension folder + removing cpptools from CashedExtensionVSIXs

![bild](https://user-images.githubusercontent.com/18281488/109550643-da0c7300-7acf-11eb-9774-9ac06bfa8dcf.png)

Do I need to try a complete reinstall of VSC now or what should I do? Clear out some other folder? Thanks for the help!
